app/layout.tsx

```diff
@@ -4,7 +4,7 @@ import {Inter} from 'next/font/google'
 
 import Header from '@/components/ui/header'
 import React from 'react';
-
+import { AppRouterCacheProvider } from '@mui/material-nextjs/v13-appRouter';
```

Comment:
Hey @Aditya-eddy I don't think it is required can't you directly use mui component in your section?

Reply:
fixed it!